lib/util.js

```diff
@@ -23,10 +23,7 @@ function longerThan(len) {
 }
 
 function getUserId(userId) {
-  if (!userId) {
-    return null
-  }
-  return userId
+  return !userId ? null : userId
```

Comment:
```suggestion
  return userId || null
```

Also the following function could be changed like this

Reply:
I've made the same fix for getClientType